packages/react-native-win32/overrides.json

```diff
@@ -330,6 +330,14 @@
       "baseHash": "31687eefb91682d95abba47f8288f7c42158706e",
       "issue": "LEGACY_FIXME"
     },
+    {
+      "type": "patch",
+      "file": "src/Libraries/LogBox/UI/LogBoxInspectorHeader.win32.js",
+      "baseFile": "Libraries/LogBox/UI/LogBoxInspectorHeader.js",
+      "baseVersion": "0.63.2",
+      "baseHash": "a69f43f63a365875af68538ed95080c537597266",
+      "issue": 5886
```

Comment:
Could you give this a unique issue (or just point to a PR with description) instead of reusing one that explains a different need to stub?